conwaysfirst.js

```diff
@@ -60,3 +44,60 @@ export function checkSquare(grid, initY, initX) {
   const output = getCellMessage(current, aliveNeighbors);
   return output;
 }
+
+export function checkAllSquares(initGrid) {
+  const Response = [];
+
+  for (let y = 0; y < initGrid.length; y++) {
+    for (let x = 0; x < initGrid[0].length; x++) {
+      const result = checkSquare(initGrid, y, x);
+      Response.push(result);
+    }
+  }
+
+  return Response;
```

Comment:
this shouldn't have a capital R as that denotes a class or module  not a variable